zgw/objecttypen-api/src/main/kotlin/com/ritense/objecttypenapi/client/ObjecttypenApiClient.kt

```diff
@@ -46,4 +46,26 @@ class ObjecttypenApiClient(
 
         return result?.body!!
     }
+
+    fun getObjecttypes(
+        authentication: ObjecttypenApiAuthentication,
+        objecttypesUrl: URI
+    ): List<Objecttype> {
+        val url = if (objecttypesUrl.host == "host.docker.internal") {
+            URI.create(objecttypesUrl.toString().replace("host.docker.internal", "localhost"))
+        } else {
+            objecttypesUrl
+        }
+        val result = webclientBuilder
+            .clone()
+            .filter(authentication)
+            .build()
+            .get()
+            .uri(url)
+            .retrieve()
+            .toEntityList<Objecttype>()
+            .block()
+
+        return result?.body!!
```

Comment:
I know this is done the same way here, but can you add some log line when the body is in fact not present? I think the additional logging is very welcome in case this goes wrong.

Reply:
What about this 

`result?.statusCode?.isError?.let {
            throw RuntimeException("Error while fetching objecttypes: ${result.statusCode}")
        }`